There issues which needs to be addressed:

```bash
************* Module homeassistant.components.twilio
homeassistant/components/twilio.py:54:8: E0611: No name 'Response' in module 'twilio.twiml' (no-name-in-module)
************* Module homeassistant.components.notify.twilio_call
homeassistant/components/notify/twilio_call.py:45:8: E0611: No name 'TwilioRestException' in module 'twilio' (no-name-in-module)
```

--- a/homeassistant/components/notify/twilio_call.py
+++ b/homeassistant/components/notify/twilio_call.py
@@ -42,7 +42,7 @@ class TwilioCallNotificationService(BaseNotificationService):
 
     def send_message(self, message="", **kwargs):
         """Call to specified target users."""
-        from twilio import TwilioRestException
+        from twilio.base.exceptions import TwilioRestException
 
         targets = kwargs.get(ATTR_TARGET)
 

--- a/homeassistant/components/twilio.py
+++ b/homeassistant/components/twilio.py
@@ -10,7 +10,7 @@ import homeassistant.helpers.config_validation as cv
 from homeassistant.core import callback
 from homeassistant.components.http import HomeAssistantView
 
-REQUIREMENTS = ['twilio==5.7.0']
+REQUIREMENTS = ['twilio==6.19.1']
 
 DOMAIN = 'twilio'
 
@@ -51,8 +51,8 @@ class TwilioReceiveDataView(HomeAssistantView):
     @callback
     def post(self, request):  # pylint: disable=no-self-use
         """Handle Twilio data post."""
-        from twilio.twiml import Response
+        from twilio.twiml import TwiML
         hass = request.app['hass']
         data = yield from request.post()
         hass.bus.async_fire(RECEIVED_DATA, dict(data))
-        return Response().toxml()
+        return TwiML().to_xml()

--- a/requirements_all.txt
+++ b/requirements_all.txt
@@ -1475,7 +1475,7 @@ transmissionrpc==0.11
 tuyapy==0.1.3
 
 # homeassistant.components.twilio
-twilio==5.7.0
+twilio==6.19.1
 
 # homeassistant.components.sensor.uber
 uber_rides==0.6.0

